### ISSUE TITLE
Switch to ghcjs 8.10

### ISSUE THREAD

**Issue by deemp** [rzk-lang/rzk#170]
It seems to be possible.

https://github.com/serokell/nixfmt/blob/422b1aebb44c4f12af025eda0ee6a99c6e50d738/flake.nix#L48-L59

**Comment by fizruk** [comment]
But why switch to an older GHC?

**Comment by deemp** [comment]
I agree, we can try to switch to something newer. Updated the issue descrition.

**Comment by deemp** [comment]
Superseded by https://github.com/rzk-lang/rzk/issues/213
